SU-SWS/stanford_basic
# D8CORE-2490: Alert Component Updates

# READY FOR REVIEW 

# Summary
- Updates to NPM Packages
- Updates to Alerts

# Review By (Date)
- Sept 3rd

# Urgency
- 10/10

# Steps to Test

1. Review code
2. Do a full build of cardinalsites or lelandd8 on the D8CORE-1609 branch of their repo
3. Clear all caches through the UI by logging in and using the admin menu links
4. Review style of new status (blue) alert
5. Click on the dismiss button
6. See alert go away
7. Navigate to `/patterns/alert` and see each alert style

Run a build of this branch. 
```
nvm use
npm install
npm run publish
```

Validate no changes to repo css.

# Affected Projects or Products
- D8CORE 
- SOE
- Adapt

# Associated Issues and/or People
- D8CORE-1609
- D8CORE-2490

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
